### PR TITLE
feat: compatible with HTTP status of `admin api`

### PR DESCRIPTION
**Pull request by nic-chen**
Please answer these questions before submitting a pull request

- Why submit this pull request?
- [ ] Bug fix
- [x] New feature provided
- [ ] Improve performance

___

### CODE REVIEW

**Review by juzhiyuan — APPROVED** [review]
Not sure if logics are ok, but looks good to me from codes style.

**Review by moonming — CHANGES_REQUESTED** [review]
No test cases

File: api/internal/core/store/validate.go [hunk 1]
```diff
@@ -125,17 +125,17 @@ func (v *APISIXJsonSchemaValidator) Validate(obj interface{}) error {
...
 			s, err := gojsonschema.NewSchema(gojsonschema.NewStringLoader(schemaDef))
 			if err != nil {
-				return fmt.Errorf("new schema failed: %w", err)
+				return fmt.Errorf("scheme validate failed: %w", err)
 			}
 
 			ret, err := s.Validate(gojsonschema.NewGoLoader(pluginConf))
 			if err != nil {
-				return fmt.Errorf("validate failed: %w", err)
+				return fmt.Errorf("scheme validate failed: %w", err)
```
Comment (liuxiran) [inline api/internal/core/store/validate.go:138]: I found that the same error messages repeatedly appeared in the files, would it be better to  manage error messages in one place just like [error.go](https://github.com/apache/apisix-dashboard/blob/master/api/errno/error.go) before :)?

**Comment by membphis** [comment]
> No test cases

https://github.com/apache/apisix-dashboard/pull/563/commits/580b88e10edd82d4c02e3d583ce56aff4123f1d1

@moonming you missed them

File: api/internal/core/store/validate.go (hunk 1 above)
Comment (ShiningRush) [reply api/internal/core/store/validate.go:138]: Agree, but we can not do it in this version for now. When the first rc version is released, we will optimize it in subsequent versions

**Review by ShiningRush — APPROVED** [review]

File: api/internal/core/store/validate.go (hunk 1 above)
Comment (liuxiran) [reply api/internal/core/store/validate.go:138]: +1 @ShiningRush

**Review by liuxiran — APPROVED** [review]

**Comment by juzhiyuan** [comment]
Let's merge it now

**Pull request merged**